doc/groups.md

```diff
@@ -1 +1,519 @@
 # Groups Module
+Module | HTTP Method | URL | Description 
+--- | --- | --- | ---
+[Add Groups](#add) | POST | /groups | Add Groups API
+[Edit Groups](#edit) | PUT | /groups/{id} | Edit Groups API
+[View All Groups](#view) | GET | /groups | View Groups API
+[Delete Groups](#delete) | DELETE | /groups/{id} | Delete Groups API
+
```

Comment:
[View](#view-by-id) | GET | /groups/{id} | View group by id

Reply:
there are 2 endpoints for view:
- view list : GET /groups
- view detail by id : GET /groups/{id}

For now only 1 endpoint in your module, just GET /groups

Reply:
okay, I'll add